### PR TITLE
chore: bump drawbridge to 0.2

### PR DESCRIPTION
**Pull request by bstrie**
Update to a version of drawbridge that contains https://github.com/profianinc/drawbridge/pull/241 , in order to fix building Enarx on certain platforms. In order to use this fix during the last release we were forced to temporarily depend on a fork of drawbridge; such workarounds should not be necessary in the future as long as the drawbridge crate permissions on crates.io are broadened appropriately.

Signed-off-by: bstrie <865233+bstrie@users.noreply.github.com>

### CODE REVIEW

**Comment by bstrie** [comment]
Note that if we're doing a 0.6.1 release, you may want to hold off on approving this unless you want to also require a new release of drawbridge.

**Review by platten — APPROVED** [review]

**Review by rvolosatovs — APPROVED** [review]

**Pull request merged**
